### ISSUE TITLE
input-req-label flags error on hidden and button inputs

### ISSUE THREAD

**Issue by nickyblissAviva** [linthtml/linthtml#297]
We have the rule input-req-label turned on but after updating from v0.4.2 to 0.6.0 it is now flagging that we have missing labels for both hidden and button inputs eg

```html
<input name="_csrf" value="{{ csrfToken }}" type="hidden" />
<span class="a-button a-button--transparent a-button--form a-button--input"><input class="a-button__inner" type="button" value="Transparent button" /></span>
```

I would not expect either of these types of input to require a label so should not be flagged.

**Comment by KamiKillertO** [comment]
Thanks for reporting I'll have a look

**Comment by KamiKillertO** [comment]
I've managed to reproduce this issue with LintHTML v0.4.2 😞 . This rule just doesn't deal with hidden and button input 🤦 . This is an inheritance from HTMLLint, I've never updated this rule 😓 . I'll fix this issue as soon as possible ^^

**Comment by KamiKillertO** [comment]
@nickyblissAviva  I've just published a new version with a fix for your problem.

**Comment by nickyblissAviva** [comment]
Thank you @KamiKillertO Once you have officially released the patch I'll update our npm package and give it a go.

**Comment by KamiKillertO** [comment]
@nickyblissAviva the patched version is already available on npm.

**Comment by nickyblissAviva** [comment]
@KamiKillertO It wasn't updating earlier. Maybe something to do with our proxy, but its updated now.

Your fix has cleared the majority of errors but I have one flagged for

`<input type="submit" value="Generate" />`

**Comment by KamiKillertO** [comment]
🤦 I forgot the submit input 🤦
